examples/nlg_server/nlg_server.py

```diff
@@ -46,20 +56,13 @@ def generate_response(nlg_call, domain):
         template, tracker, channel_name, **kwargs)
 
 
-def create_app(domain):
-    app = Flask(__name__)
-
-    logging.basicConfig(level=logging.DEBUG)
-
-    @app.route("/nlg", methods=['POST', 'OPTIONS'])
-    def nlg():
-        """Check if the server is running and responds with the version."""
-        nlg_call = request.json
+@app.route("/nlg", methods=['POST', 'OPTIONS'])
+async def nlg(request):
+    """Endpoints which processes the Core request for a bot response."""
+    nlg_call = request.json
+    response = await generate_response(nlg_call, domain)
 
-        response = generate_response(nlg_call, domain)
-        return jsonify(response)
-
-    return app
+    return json(response)
```

Comment:
lets use the package (as their are other jsons's around)
```suggestion
    return response.json(response)
```